src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -224,9 +225,24 @@ public final boolean load(SectionNode sectionNode) {
 			// Set the backup interval, if present & enabled
 			if (!"0".equals(getValue(sectionNode, "backup interval"))) {
 				Timespan backupInterval = getValue(sectionNode, "backup interval", Timespan.class);
-
+				int tokeep = getValue(sectionNode, "backups to keep", Integer.class);
+				boolean purge = false;
+				boolean proceed = true;
 				if (backupInterval != null)
-					startBackupTask(backupInterval);
+					if (tokeep == 0) {
+						proceed = false;
+					} else if (tokeep >= 1) {
+						purge = true;
+					}
+					if (proceed) {
+                        startBackupTask(backupInterval, purge, tokeep);
+                    } else {
+						try {
+							FileUtils.backupPurge(file, tokeep);
```

Comment:
The naming schema here's a bit confusing, with the various meanings of purge, the ambiguity of proceed, etc. This should be improved for clarity.